nsv13/code/controllers/subsystem/starsystem.dm

```diff
@@ -953,19 +953,22 @@ Returns a faction datum by its name (case insensitive!)
 		var/roid_type = pick(/obj/structure/overmap/asteroid, /obj/structure/overmap/asteroid/medium, /obj/structure/overmap/asteroid/large)
 		SSstar_system.spawn_ship(roid_type, src)
 
-/datum/star_system/proc/spawn_enemies(enemy_type, amount)
-	if(!amount)
-		amount = difficulty_budget
-		if(amount <= 0)
-			amount = 1 //Why else are you calling this?
-	for(var/i = 0, i < amount, i++) //number of enemies is set via the star_system vars
-		if(!enemy_type)
-			enemy_type = pick(SSstar_system.enemy_types) //Spawn a random set of enemies.
-		SSstar_system.spawn_ship(enemy_type, src)
+/datum/star_system/proc/spawn_enemies(difficulty) //spawns a
+	if(!difficulty)
+		difficulty = round(difficulty_budget*1.5) //On average these spawn a bit easier, but they can scale
+		if(difficulty <= 0)
+			difficulty = 1 //Why else are you calling this?
+	var/fleet_type
+	if(alignment == owner && alignment == "syndicate") //defending their system
+		fleet_type = /datum/fleet/light_defense
+	else
+		var/datum/faction/FC = SSstar_system.faction_by_name("syndicate")
+		fleet_type = pick(FC.fleet_types)
+	spawn_fleet(fleet_type, difficulty)
```

Comment:
This is going to make the random garrisons in systems significantly more dangerous since instead of a few random ships, you will always get at least the minimum fleet composition.
In addition to that, due to the scaling / reinforcement system, you *may* also get fairly powerful forces regardless of the initial difficulty, depending on how fast the game initializes, but only sometimes (reinforce depends on the move proc which defense trait fleets only ever call once, which may not do anything depending on init delays).

Not entirely sure how I feel about them since these also stack with the fleets you are probably in the system for.

Reply:
I did set out to make the syndicate systems more dangerous/challenging than they were with just two random ships, in general.
This was also with the idea they couldn't scale more than the roundstart difficulty because they don't move.

It's also just nicer to have them count as fleets, even if they're not very strong, because defeating a handful of ships and them not counting as a fleet for objectives (or tickets) just kind of sucked.

All in all I haven't heard a complaint about them so far, which makes me think they're doing alright now.

Reply:
Still somewhat on the fence about them due to fleet density but eh, something to keep an eye on.